src/test/java/UhrzeitenTest.java

```diff
@@ -18,23 +17,10 @@ public void assembleAllValidTimes() {
 		assertThat(times.size(), not(2000));
 	}
 
-	@Test(expected = IndexOutOfBoundsException.class)
+	@Test(expectedExceptions = IndexOutOfBoundsException.class)
```

Comment:
As you can see, we now no longer use the testing framework JUnit but TestNG. This offers us much more features concerning permuting tests. However it has a slightly other syntax. You noticed the difference in expecting exceptions. Watch the changed import...

`import org.testng.annotations.Test;`